docs/input.md

```diff
@@ -110,6 +110,10 @@ The situation is different for certain models:
 
 - [`AnthropicModel`][pydantic_ai.models.anthropic.AnthropicModel]: if you provide a PDF document via `DocumentUrl`, the URL is sent directly in the API request, so no download happens on the user side.
 
+- [`GoogleModel`][pydantic_ai.models.google.GoogleModel] on GLA:
+    - YouTube video URLs are sent directly in the request to the model.
+    - Files uploaded to the [Files API](https://ai.google.dev/gemini-api/docs/files) (by prefix matching on `https://generativelanguage.googleapis.com/v1beta/files`)
+
```

Comment:
you can remove the parentheses after the link, there should also be some wording consistent with `Anthropic` and `GoogleModel on Vertex`

"Files uploaded to the Files API ... can be passed via their URL", or something like that